client/lib/plans/constants.js

```diff
@@ -1081,13 +1083,11 @@ export const PLANS_LIST = {
 			),
 		getDescription: () =>
 			i18n.translate(
-				'The most powerful WordPress sites: unlimited premium ' +
-					'themes, real-time backups, and enhanced search.'
+				'The most powerful WordPress sites: real-time backups, ' +
+					'enhanced search, and unlimited premium themes.'
 			),
 		getTagline: () =>
-			i18n.translate(
-				'You have full access to premium themes, marketing tools, and priority support.'
-			),
+			i18n.translate( 'You have full the full suite of security and performance tools.' ),
```

Comment:
Typo here btw. :) 

```suggestion
			i18n.translate( 'You have the full suite of security and performance tools.' ),
```

Reply:
Thanks for noticing that! Fixed in https://github.com/Automattic/wp-calypso/pull/31465/commits/124f5d37220233d46c0395acbbe0e33475a52a64